### PR TITLE
Add warning to obeliskEnvs.command which caused major confusion

### PR DESCRIPTION
**Pull request by 3noch**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -106,7 +106,9 @@ in rec {
   pathGit = ./.;  # Used in CI by the migration graph hash algorithm to correctly ignore files.
   path = reflex-platform.filterGit ./.;
   obelisk = ghcObelisk;
-  obeliskEnvs = ghcObeliskEnvs;
+  obeliskEnvs = ghcObeliskEnvs // {
```
Comment (matthewbauer) [inline default.nix]: Maybe we should just filter ghcObelisk for just the `obelisk-*` attrs? I'm not sure whether we have similar collisions like this.

Comment (3noch) [reply default.nix]: Good idea

**Comment by ingenieroariel** [comment]
@Ericson2314 what are your thoughts on this?

File: default.nix (hunk 1 above)
Comment (3noch) [reply default.nix]: I did this instead.

**Pull request merged**